### PR TITLE
 Update Portuguese translations for CJB Cheats Menu and CJB Item Spawner

### PR DESCRIPTION
**Pull request by MagoSupremo123**

### CODE REVIEW

File: CJBCheatsMenu/i18n/pt.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "tabs.player-and-tools": "Jogador e Ferramentas",
     "tabs.farm-and-fishing": "Cultivo e Pesca",
     "tabs.skills": "Habilidades",
-    "tabs.weather": "Tempo",
```
Comment (MagoSupremo123) [inline CJBCheatsMenu/i18n/pt.json:12]: "Tempo" in portuguese means both time and weather, so this should make it more clear

File: CJBCheatsMenu/i18n/pt.json [hunk 2]
```diff
@@ -121,9 +121,9 @@
     /*********
     ** Weather tab
     *********/
-    "weather.title": "Tempo Amanhã",
```
Comment (MagoSupremo123) [inline CJBCheatsMenu/i18n/pt.json:124]: Same thing

**Review by MagoSupremo123 — COMMENTED** [review]
I beyond that, no further comment from me is necessary.

**Review by Pathoschild — APPROVED** [review]

**Comment by Pathoschild** [comment]
Merged into `develop` for the upcoming CJB Cheats Menu 1.37.0 and CJB Item Spawner 2.4.2. Thanks for the help!

**Pull request merged**